apps/modernization-ui/src/design-system/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,36 @@
+import { Pagination as PaginationUswds } from '@trussworks/react-uswds';
+import styles from './pagination.module.scss';
+import { RangeToggle } from 'components/Table/RangeToggle/RangeToggle';
+import { useEffect } from 'react';
+import { usePage } from 'page';
+
+type Props = {
+    total: number;
+    pageSize: number;
```

Comment:
issue:  The total and pageSize should really come from the PageState provided by `usePage`.  The calculation for the total number of pages is not updating correctly because it's only ever doing result total / 10 as the page size passed into the component is never changing.